json-rpc/docs/method_get_account_transaction.md

```diff
@@ -1,5 +1,7 @@
 ## Method get_account_transaction
 
+// Why does this exist?  It's the same as get account transactions with limit 1
```

Comment:
it existed before get account transactions, this is more handy for get specific transaction, especially wait for transaction executed.